src/components/ChooseProgram.jsx

```diff
@@ -1,15 +1,12 @@
-import Button from "./buttons/ButtonPrimary";
-
-// TODO zentrieren
+import ButtonPrimary from "./buttons/ButtonPrimary";
```

Comment:
Instead of a button, we suggest an anker tag because it only links to the page "Browse Programs" and does not manipulate something on the dashboard page

Reply:
ups, yes, I always get this wrong, no matter how often I put it down somewhere as "remember, button for action, anker for navigation" 

the centering could also be done while calling the button, I guess we have only centered ones, but just in case...

Reply:
hhm, but when I turn it into a <div><a href="">Buttontext</a></div> it looses the styling that it had so neatly when it was a button

question for Eric :-D